activerecord/lib/active_record/nested_attributes.rb

```diff
@@ -280,6 +280,26 @@ class TooManyRecords < ActiveRecordError
     #   member = Member.new
     #   member.avatar_attributes = {icon: 'sad'}
     #   member.avatar.width # => 200
+    #
+    # === Forms
```

Comment:
Maybe expand the title a bit?
```suggestion
    # === Creating forms with nested attributes
```